examples/ask-the-documents/src/components/DocumentCard.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from "@heroui/react";
 import { searchDocuments } from "wasp/client/operations";
 
-import ReactMarkdown from "react-markdown";
+import Markdown from "react-markdown";
```

Comment:
They use just `Markdown` in their examples: https://github.com/remarkjs/react-markdown/blob/main/changelog.md#remove-classname